feature/home/presentation/home-ui/src/main/java/com/chocolatecake/ui/common/adapters/MediaVerticalAdapter.kt

```diff
@@ -2,12 +2,12 @@ package com.chocolatecake.ui.common.adapters
 
 import com.chocolatecake.bases.BaseAdapter
 import com.chocolatecake.ui.home.R
-import com.chocolatecake.viewmodel.common.listener.MediaListener
 import com.chocolatecake.viewmodel.common.model.MediaVerticalUIState
+import com.chocolatecake.viewmodel.people.PeopleDetailsViewModel
 
 class MediaVerticalAdapter(
     list: List<MediaVerticalUIState>,
-    listener: MediaListener
+    listener: PeopleDetailsViewModel
```

Comment:
make PeopleDetailsViewModel implement MediaListener and return it to MediaListener